plugins/invisible-typing/src/index.ts

```diff
@@ -12,7 +12,7 @@ export interface SettingsType {
   button?: boolean;
   invisible?: boolean;
   channelWise?: boolean;
-  channels?: Record<string, boolean>;
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Allowing undefined values in channels record aligns with runtime behavior.

```suggestion
  channels?: Partial<Record<string, boolean>>;
```